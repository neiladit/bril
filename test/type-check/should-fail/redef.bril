main{

    v0: int = const 5;
    v0: bool = const true;
    print v0;

}
