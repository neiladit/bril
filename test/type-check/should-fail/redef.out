line 3: re-assigning int value to existing bool variable
const error!
